aws-lc-rs/src/rsa/key.rs

```diff
@@ -325,6 +337,14 @@ impl AsRef<[u8]> for PublicKey {
     }
 }
 
+impl AsDer<PublicKeyX509Der<'static>> for PublicKey {
+    fn as_der(&self) -> Result<PublicKeyX509Der<'static>, Unspecified> {
+        // TODO: refactor
```

Comment:
What's the plan to refactor?

Reply:
Currently the `PublicKey` struct only holds the serialized (RFC8017) bytes for the public key.  I would like for `PublicKey` to instead hold a reference to the EVP_PKEY, from which we could perform this and other operations.